### DISCUSSION TITLE
Optionen bei der Tourenplanung

### DISCUSSION THREAD

**Discussion by 1maxi2** [devemux86/cruiser#60] [Q&A] [answered]
Hello
In kurviger gab es die Optionen bei der Planung : Extra kurvige Route,
kurvige Route, schnellste Route usw. 
Das war sehr praktisch. Meist war mit Auswahl " kurvige Route" schnell eine Tagestour gemacht ohne das mit Wegpunkten " nachgearbeitet werden musste.
Ist diese Option auch in Cruiser geplant ?
Lg.Helmut

**Comment by devemux86** [comment, marked as answer]
Hi @1maxi2!

Cruiser can **import / export kurviger routes offline** (not use kurviger routing).

Navigation rerouting can be offline and only changes the part from user location to next waypoint.
_The rest of the curvy route remains unchanged._

---

Hallo @1maxi2!

Cruiser kann **kurvige Routen offline importieren / exportieren** (nicht kurviges Routing verwenden).

Das Navigations-Rerouting kann offline erfolgen und ändert nur den Teil vom Benutzerstandort zum nächsten Wegpunkt.
_Der Rest der kurvigen Route bleibt unverändert._

**Reply by 1maxi2** [reply]
Hello
Danke für die rasche Antwort.
Aber ich meinte diese Optionen wie im Bild zu sehen
Lg
![Screenshot_20230225_121313_Kurviger Pro](https://user-images.githubusercontent.com/124886589/221353879-d970b0cb-b69c-4720-a7f7-b9f0ef3daf9c.jpg)

**Reply by devemux86** [reply]
These options are for Kurviger **Routing** (not available in Cruiser).

Only **import / export of kurviger routes** is available in Cruiser.

---

Diese Optionen sind für den Kurviger **Routing** (nicht verfügbar im Cruiser).

Nur der **Import / Export von Kurviger-Routen** ist im Cruiser verfügbar.

**Reply by 1maxi2** [reply]
O.k. kein großes Problem da mit ein paar gesetzten Wegpunkten  schnell eine Route geplant ist.
LG.

**Reply by devemux86** [reply]
You can import your Kurviger routes offline as *.kurviger files (export them from the website).
And navigation rerouting does not change the whole route (only the part until next waypoint).

---

Sie können Ihre Kurviger Routen offline als *.kurviger Dateien importieren (exportieren Sie sie von der Website).
Und die Navigationsumleitung ändert nicht die gesamte Route (nur den Teil bis zum nächsten Wegpunkt).
